### PR TITLE
Feature/yaml settings

### PR DESCRIPTION
**Pull request by matthewware**
Could people take this for a spin?  @gribeill @grahamrow

### CODE REVIEW

**Comment by gribeill** [comment]
@mware I fixed some path stuff but there's one test (in `test_hdf5write`) this is still failing... it doesn't seem to be writing the header?

**Comment by mware** [comment]
@gribeill I think you meant @matthewware

File: src/auspex/filters/io.py [hunk 1]
```diff
@@ -152,6 +173,13 @@ def save_json(self):
 
         self.data_group = self.group.create_group("data")
 
+        # If desired, push experimental metadata into the h5 file
```
Comment (grahamrow) [inline src/auspex/filters/io.py]: Do we even want the option of saving JSON at this point?

File: src/auspex/filters/io.py [hunk 2]
```diff
@@ -18,6 +18,7 @@
 import time
 import re
 from shutil import copyfile
+from ruamel.yaml import YAML
```
Comment (grahamrow) [inline src/auspex/filters/io.py]: Does this get used here?

File: src/auspex/filters/io.py (hunk 1 above)
Comment (dieris) [reply src/auspex/filters/io.py]: I don't think so

**Pull request merged**